examples/lfric/eg2/print_psyir_trans.py

```diff
@@ -1,7 +1,7 @@
 # -----------------------------------------------------------------------------
 # BSD 3-Clause License
 #
-# Copyright (c) 2017-2021, Science and Technology Facilities Council.
+# Copyright (c) 2017-2021, Science and Technology Facilities Council
```

Comment:
Erm, I think full stop to be consistent with the places where it is added 😄

Reply:
Oh yes, don't know why I did that :-)